tests/test_enterprise/test_utils.py

```diff
@@ -97,14 +98,15 @@ def test_get_platform_logo_url(self, logo_url, expected_logo_url, mock_get_logo_
         mock_get_logo_url.return_value = logo_url
         self.assertEqual(get_platform_logo_url(), expected_logo_url)
 
-    @mock.patch('enterprise.utils.lms_enroll_user_in_course')
+    @override_settings(ENABLE_ENTERPRISE_BACKEND_EMET_AUTO_UPGRADE_ENROLLMENT_MODE=True)
+    @mock.patch('enterprise.utils.lms_update_or_create_enrollment')
```

Comment:
[suggestion] Similar to the previous comment, I wonder if we should have test cases covering both the feature flag being enabled and disabled to assert on both code baths while both still exist.